nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java

```diff
@@ -213,9 +217,45 @@ public static String serializeSymbol(@Nullable Symbol symbol) {
         return symbol.name.toString();
       case METHOD:
       case CONSTRUCTOR:
-        return symbol.toString();
+        return serializeMethodSignature((Symbol.MethodSymbol) symbol);
       default:
         return symbol.flatName().toString();
     }
   }
+
+  /**
+   * Serializes the signature of the given {@link Symbol.MethodSymbol} to a string. This code is
+   * based on {@link com.google.errorprone.util.Signatures#prettyMethodSignature}}.
```

Comment:
You have a point, this code is pretty different from the Error Prone code now.  So probably we can remove the sentence referencing the Signatures function.  Sorry to go back and forth :-)

Reply:
(Agreed on this one!)

Reply:
Sure, removed the reference 6e1406b046198fa5762611d51ac9be249dd785aa